core/verify_modes.go

```diff
@@ -0,0 +1,69 @@
+package core
```

Comment:
recommend to integrate this file into verify_manager.go.

Reply:
I think so, it is not proper to be a single file under `core` package.